extensions/cache/runtime-spi/src/main/java/io/quarkus/cache/Cache.java

```diff
@@ -55,6 +56,16 @@ public interface Cache {
      */
     <K, V> Uni<V> getAsync(K key, Function<K, Uni<V>> valueLoader);
 
+    /**
+     * Modify the cache entry identified by {@code key} from the cache.
+     * The command function get passed both the key and the previous value, and should return the new value.
+     *
+     * @param key cache key
+     * @param command function having for argument the key and the previous value if any, returning the new value
+     * @throws NullPointerException if the key is {@code null}
```

Comment:
Shouldn't it be an IllegalArgumentException?

Reply:
NullPointerException is consistent with the rest of the API, but IllegalArgumentException make more sense in my opinion.
Maybe this can be done in another PR ?